.github/ISSUE_TEMPLATE/test-failure.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Issue Template for Reporting Testing Failures
+about: Describe the failure you encountered while testing
+title: "[Test Case ID]: descriptive title"
+labels: ''
+assignees: ''
```

Comment:
Can we here at labels section make something like 
`labels: <type release name here> testings`

Reply:
I don't know what this comment means. Are you hoping for a dropdown menu of suggested labels/releases?

Reply:
I am not sure how github UX/UI works, but yeah that can be dropdown list or something... _the is if we are sure that each test issue should have a label crossponding to it's release test_ . Then wouldn't make sense to make assigning of that label automated

Reply:
Do you think it's worth it to automatically categorize it as `release testing`?

Reply:
I wouldn't do a dropdown of labels unless you're committing to keeping labels & this template in sync.

Maybe initially it's `needs triage`?

Reply:
That one sounds good